hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/protocolPB/OMPBHelper.java

```diff
@@ -242,11 +242,20 @@ public static MD5MD5Crc32FileChecksumProto convert(
     DataOutputBuffer buf = new DataOutputBuffer();
     checksum.write(buf);
     byte[] bytes = buf.getData();
-    DataInputBuffer buffer = new DataInputBuffer();
-    buffer.reset(bytes, 0, bytes.length);
-    int bytesPerCRC = buffer.readInt();
-    long crcPerBlock = buffer.readLong();
-    buffer.close();
+    DataInputBuffer buffer = null;
+    int bytesPerCRC;
+    long crcPerBlock;
+    try {
+      buffer = new DataInputBuffer();
+      buffer.reset(bytes, 0, bytes.length);
+      bytesPerCRC = buffer.readInt();
+      crcPerBlock = buffer.readLong();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    } finally {
+      assert buffer != null;
+      buffer.close();
+    }
```

Comment:
Let's rely on auto-close:

```suggestion
    int bytesPerCRC;
    long crcPerBlock;
    try (DataInputBuffer buffer = new DataInputBuffer()) {
      buffer.reset(bytes, 0, bytes.length);
      bytesPerCRC = buffer.readInt();
      crcPerBlock = buffer.readLong();
    }
```